app/src/main/res/layout/covid_layout.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <variable
```

Comment:
In the future you could have a Covid object variable inside your **CovidViewModel**. This way you only use the viewmodel as a variable inside this xml layout. Once these changes are made you need to notify to the binding object that the Covid object has changed everytime you make a new call to the API

Reply:
I added the necessary variables to display data (to be exact 3 variables. date, confirmedCases, deaths) in the viewModel. I didn't add the Covid object.

Reply:
It would be nice create and use the object that Dan mentioned above, because in that way the code is cleaner